spyder/widgets/fileswitcher.py

```diff
@@ -655,21 +656,29 @@ def setup_file_list(self, filter_text, current_path):
         self.fix_size(paths)
 
         # Build the text that will appear on the list widget
+        path_text_font_size = CONF.get('appearance', 'rich_font/size', 11)
+        filename_text_font_size = path_text_font_size + 2
```

Comment:
Why did you use `+ 2` here instead of `+ 1`? Is there too little difference with only one point between filename and path_text_font_size and filename_text_font_size?

Reply:
@juanis2112? or @dalthviz?

Reply:
Yep, the difference seemed too little

Reply:
![imagen](https://user-images.githubusercontent.com/16781833/51999965-f15c7a80-2489-11e9-9432-59c8ab92dbfa.png)

or what do you think?